### PR TITLE
Finish quarantine for test_should_force_kill_process

### PR DESCRIPTION
**Pull request by potiuk**
Changing the test to check actual PID of the process to kill

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: tests/utils/test_process_utils.py [hunk 1]
```diff
@@ -136,23 +136,21 @@ def test_should_kill_process(self):
...
-        before_num_process = subprocess.check_output(["ps", "-ax", "-o", "pid="]).decode().count("\n")
 
         process = multiprocessing.Process(target=my_sleep_subprocess_with_signals, args=())
         process.start()
         sleep(0)
 
-        num_process = subprocess.check_output(["ps", "-ax", "-o", "pid="]).decode().count("\n")
-        assert before_num_process + 1 == num_process
+        all_processes = subprocess.check_output(["ps", "-ax", "-o", "pid="]).decode().splitlines()
+        assert str(process.pid) in map(lambda x: x.strip(), all_processes)
```
Comment (uranusjr) [inline tests/utils/test_process_utils.py:146]: Would this work?

```suggestion
        all_processes = subprocess.check_output(["ps", "-ax", "-o", "pid="]).decode().splitlines(keepends=False)
        assert str(process.pid) in all_processes
```

Comment (potiuk) [reply tests/utils/test_process_utils.py:146]: That would not work. The returned values do not have EOL. They are prefixed with spaces: 

```
       1
       2
  234333
```

Comment (uranusjr) [reply tests/utils/test_process_utils.py:146]: Ouch. Thanks for the explanation.

**Comment by potiuk** [comment]
very small - removes a test from quarantine

**Comment by potiuk** [comment]
Anyone :)

**Review by turbaszek — APPROVED** [review]

**Pull request merged**